.NET Core run fails with missing module exception

**I'm submitting a bug report**

* **Library Version:**
1.0.0-beta.14


**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
10.15.3

* **Yarn Version:**
1.13.0

* **Browser:**
all 
* **Language:**
all 

* **Loader/bundler:**
Webpack

**Current behavior:**
For the .Net Core platform, the _clean-webpack-plugin_ is now missing, so running the project 
with the `dotnet run` command produces an exception on startup.

Create new project with the following options:
√ Would you like to use the default setup or customize your choices? · **Custom App**
√ Which bundler would you like to use? · **Webpack**
√ Which HTTP Protocol do you wish the outputted Webpack bundle to be optimised for? · **HTTP/1.1**
√ What platform are you targeting? · **.NET Core**
√ What transpiler would you like to use? · **TypeScript**
√ How would you like to setup your HTML template? · **None**
√ What css processor would you like to use? · **None**
√ Do you want to add PostCSS processing · **None**
√ Which unit test runner would you like to use? · **None**
√ Would you like to configure integration testing? · **None**
√ What is your default code editor? · **Visual Studio Code**
√ Which features do you want scaffolded into your project? · **Navigation App**

Run `dotnet run`

Fails with exception:

> Application startup exception: System.AggregateException: One or more errors occurred. (Cannot find module 'clean-webpack-plugin'
> Error: Cannot find module 'clean-webpack-plugin'

If I add the `"clean-webpack-plugin": "^0.1.19"` back to the devDependencies as it was in beta13, the project runs normally.


* **What is the expected behavior?**
<!--
If the current behavior is a bug, please provide the steps to reproduce.
-->
Project runs without error


* **What is the motivation / use case for changing the behavior?**
This is a bug, a new project should run out of the box with no modification.

--- a/lib/workflow/questionnaire.js
+++ b/lib/workflow/questionnaire.js
@@ -197,7 +197,7 @@ exports.askScaffold = {
   }, {
     value: 'scaffold-navigation',
     message: 'Navigation App',
-    hint: 'Add a router and some sample routes, Bootstrap v4 and Font Awesome v4.'
+    hint: 'Add a router and some sample routes, Bootstrap v3 and Font Awesome v4.'
   }]
 };
 

--- a/skeleton/dotnet-core/package.json__if_webpack
+++ b/skeleton/dotnet-core/package.json__if_webpack
@@ -1,0 +1,5 @@
+{
+  "devDependencies": {
+    "clean-webpack-plugin": ""
+  }
+}

